### PR TITLE
Create expires soon page

### PR DESCRIPTION
**Pull request by ghiocgeorge**
I create a new HTML page (**expires_soon.html**). On this page will find all the items that will expire today or in 5 days. I make a new method in **dashboardController** class (_viewExpiresSoonItems_). With this method, I populated the HTML page with only items that are expiring soon, those items that are following the rules written above.

### CODE REVIEW

File: src/main/java/ro/vladutit/Don/t/forget/v2/controller/DashboardController.java [hunk 1]
```diff
@@ -60,6 +67,41 @@ public String viewDashboardAll(
...
+
+    @RequestMapping("/expires_soon")
+    public String viewExpiresSoonItems(
+            Model category,
+            Model categoryList,
+            Model item,
+            Model itemList,
+            @AuthenticationPrincipal CustomUserDetails userDetails) {
+        final String startDate = simpleDate.format(new Date());
+        final String endDate = LocalDate.parse(simpleDate.format(new Date())).plusDays(5).toString();
```
Comment (andreisamoila74) [inline src/main/java/ro/vladutit/Don/t/forget/v2/controller/DashboardController.java:95]: you shouldn't hardcode this number 5. you should have an app config, that tell you this kind of information. do you understand what I mean by app config?

**Review by andreisamoila74 — CHANGES_REQUESTED** [review]

File: src/main/java/ro/vladutit/Don/t/forget/v2/controller/DashboardController.java (hunk 1 above)
Comment (ghiocgeorge) [reply src/main/java/ro/vladutit/Don/t/forget/v2/controller/DashboardController.java:95]: Hmm, no, I didn't understand why I need this if for now, the users can't change this number of days, and also, isn't very clear what you want to say with this app config.  Maybe to create a new field in Item Model, where I saved this info, to be easy to make a new future for users in the future? Or when you say "app config" you refer to a centralized place like `application.proprieties` for Spring Boot, and the info to be taken from there? Please tell me if I understood and which approach is the best for now. Thank you!

Comment (andreisamoila74) [reply src/main/java/ro/vladutit/Don/t/forget/v2/controller/DashboardController.java:95]: something like a centralized configuration menu for admins. to set thing like this

Comment (andreisamoila74) [reply src/main/java/ro/vladutit/Don/t/forget/v2/controller/DashboardController.java:95]: it's a table in the database that contains different info for some global settings in your app

Comment (ghiocgeorge) [reply src/main/java/ro/vladutit/Don/t/forget/v2/controller/DashboardController.java:95]: Ok. Now I think I understood. I will create an issue for this task. Thank you.

**Pull request merged**